resources/web/wwi/Toolbar.js

```diff
@@ -257,6 +258,37 @@ export default class Toolbar {
       this._triggerPlayPauseButton();
   }
 
+  _createIdeButton() {
+    this.ideButton = this._createToolBarButton('ide', 'Theia IDE', undefined);
```

Comment:
Not sure about this one, maybe simply "IDE" or maybe better "Source Code Editor" would be a less specific tooltip as most people don't know about Theia...